lib/modules/manager/helmfile/artifacts.ts

```diff
@@ -44,14 +49,16 @@ export async function updateArtifacts({
   try {
     await writeLocalFile(packageFileName, newPackageFileContent);
 
+    // we need to prefix version with v to install helmfile.
+    const helmfileVersion = `v${parseLock(existingLockFileContent).version}`;
```

Comment:
is the version always available inside the lockfile? if not this will throw and cause unwanted aborts.

also remove the v prefix, renovate (containerbase install-tool) will handle it for you.

Reply:
> is the version always available inside the lockfile? if not this will throw and cause unwanted aborts.
Yes.
source: https://github.com/helmfile/helmfile/blob/c55fa0f7656a25f78c8a927a3186f7ee8617338a/pkg/state/chart_dependency.go#L54

> also remove the v prefix, renovate (containerbase install-tool) will handle it for you.

OK! i will be fix it